src/Hl7.Fhir.Core/Rest/EntryToHttpExtensions.cs

```diff
@@ -53,9 +54,17 @@ internal static class EntryToHttpExtensions
             {
                 request.Headers["Prefer"] = bodyPreference == Prefer.ReturnMinimal ? "return=minimal" : "return=representation";
             }
-
+            
             if (entry.Resource != null)
-                setBodyAndContentType(request, entry.Resource, format, CompressRequestBody, out body);
+            {
+                bool isSearch =
```

Comment:
The name of this variable put me on the wrong foot.  Shouldn't this be called  `bools isSearchUsingPost`?

Reply:
Agree, maybe lose the is-part altogether and call it searchUsingPost.